### PR TITLE
[plugin.video.mlbtv] 2018.7.11

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Description

- Added Catch Up feature to live games
- Fix Stream order bug
- List game start time instead of Pre-Game
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.mlbtv/resources/lib/mlb.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from resources.lib.globals import *
 from account import Account
```
Comment (Rechi) [inline plugin.video.mlbtv/resources/lib/mlb.py]: `from .account import Account` for py3 compatibility

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout mlb
git reset --hard upstream/krypton
git checkout origin/mlb -- plugin.video.mlbtv/
git commit -m '[plugin.video.mlbtv] 2018.7.11'
git push -f origin mlb
```

**Comment by eracknaphobia** [comment]
Ok, I ran the commands (thanks!)

**Pull request merged**